demos/asp-net-core/Divergent.Sales.API.Host/Startup.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Owin.Cors;
+using Newtonsoft.Json.Serialization;
+using Owin;
+using System.Net.Http.Formatting;
+using System.Web.Http;
+
+namespace Divergent.Sales.API.Host
+{
+    public class Startup
+    {
+        public void Configuration(IAppBuilder appBuilder)
+        {
+            var config = new HttpConfiguration();
+
+            config.Formatters.Clear();
+            config.Formatters.Add(new JsonMediaTypeFormatter());
+
+            config.Formatters
+                .JsonFormatter
+                .SerializerSettings
+                .ContractResolver = new CamelCasePropertyNamesContractResolver();
+
+            config.MapHttpAttributeRoutes();
+
+            config.Routes.MapHttpRoute(
+                name: "DefaultApi",
+                routeTemplate: "api/{controller}/{id}",
+                defaults: new { id = RouteParameter.Optional }
+            );
+
+            appBuilder.UseCors(CorsOptions.AllowAll);
+            appBuilder.UseWebApi(config);
```

Comment:
API Hosts are still full framework based, this is because in the original demo these hosts were an exact copy of the ones used in exercises, it's easier for attendees to connect the dots.
I propose that we leave them unchanged so that they are still as the exercises one and raise an issue to convert them to .NET Core, maybe in the same timeframe we convert exercises to .NET Core.